tests/integration/Router/Endpoints/ViewInBrowserTest.php

```diff
@@ -106,7 +106,7 @@ public function testItFailsValidationWhenSubscriberTokenDoesNotMatch() {
 
   public function testItFailsValidationWhenNewsletterIdIsProvidedButSubscriberDoesNotExist() {
     $data = (object)$this->browserPreviewData;
```

Comment:
Maybe in this case we could just drop conversion to object and use array as we do in API so that we don't have to add so many `// phpcs:ignore Squiz.NamingConventions.ValidVariableName.NotCamelCaps` but it is probably out of scope of this task.